database/sql/src/main/scala/cromwell/database/sql/tables/CallCachingJobDetritusEntry.scala

```diff
@@ -0,0 +1,9 @@
+package cromwell.database.sql.tables
+
+case class CallCachingJobDetritusEntry
```

Comment:
Hmm, why?

Genuine question, this makes mocking a nightmare and does it really matter if someone extends this?